src/main/java/com/direwolf20/buildinggadgets/common/integration/IBlockProviderItemHandler.java

```diff
@@ -0,0 +1,26 @@
+package com.direwolf20.buildinggadgets.common.integration;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.items.IItemHandler;
+
+public interface IBlockProviderItemHandler extends IItemHandler {
```

Comment:
I don't think this should be called `IBlockProviderItemHandler` as it has nothing to do with Blocks. It just adds an extended interface for accessing the items in the container. `IExtendedItemHandler`\`IAdvancedItemHandler` or `IItemAccess` maybe? The last is the best one of these 3 imo, as it lays the focus on it being used for accessing Items in BG.

Reply:
Other suggestions?

Reply:
I agree, the name is a bit wordy. `IItemAccess` sounds good to me.